modules/cli-manager-installing.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-installing_{context}"]
+= Installing the {cli-manager}
+
+Install the {cli-manager} to facilitate adding CLI plugins in both connected and disconnected environments.
+
+.Prerequisites
+
+* link:https://krew.sigs.k8s.io/docs/user-guide/setup/install[Krew is installed].
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Create the required namespace for the {cli-manager}.
+.. Navigate to *Administration* -> *Namespaces* and click *Create Namespace*.
+.. Enter `openshift-cli-manager-operator` in the *Name* field and click *Create*.
++
+
+. Install the {cli-manager}.
+.. Navigate to *Operators* -> *OperatorHub*.
+.. Enter *{cli-manager}* into the filter box.
+.. Select the *{cli-manager}* and click *Install*.
+.. On the *Install Operator* page:
+... The *Update channel* is set to *tech preview*, which installs the latest technology preview release of the {cli-manager}.
+... Select *A specific namespace on the cluster* and select *openshift-cli-manager-operator* from the drop-down menu.
+... Click *Install*.
++
+
+. Create the CliManager operand.
+.. Navigate to *Operators* -> *OperatorHub*.
+.. Enter *{cli-manager}* into the filter box.
+.. Select the *{cli-manager}*
+.. Select *CLI Manager*.
```

Comment:
Instead should we say navigate to `InstalledOperators->click on cli-manager` ?Then click create climanager ?